src/components/MAPI/organizations/AccessControl/AccessControlRoleSubjects.tsx

```diff
@@ -191,23 +223,64 @@ const AccessControlRoleSubjects: React.FC<IAccessControlRoleSubjectsProps> = ({
 
     try {
       dispatch({ type: 'startLoading', subjectType: type });
-      await onAdd(type, values);
+      const statuses = await onAdd(type, values);
       dispatch({ type: 'stopAdding', subjectType: type });
 
-      let message = '';
-      if (values.length > 1) {
-        message = 'Subjects added successfully.';
+      if (statuses) {
+        const createdAccountsMessage = getBindServiceAccountSuccessMessage(
+          values,
+          statuses,
+          ui.AccessControlRoleSubjectStatus.Created
+        );
+        if (createdAccountsMessage.length > 0) {
+          new FlashMessage(
+            createdAccountsMessage,
+            messageType.SUCCESS,
+            messageTTL.MEDIUM
```

Comment:
The messages are a bit longer, so I increased the length of time the flash message is displayed